openedx/core/djangoapps/credit/signals.py

```diff
@@ -27,6 +31,7 @@ def on_course_publish(course_key):  # pylint: disable=unused-argument
     # Import here, because signal is registered at startup, but items in tasks
     # are not yet able to be loaded
     from openedx.core.djangoapps.credit import api, tasks
+    tag_course_content_with_partition_scheme(course_key, partition_scheme='verification')
```

Comment:
Move this line after the `if api.is_credit_course(course_key):` this check so that the task is generated for parallel processing.